pkg/cmd/root/help.go

```diff
@@ -302,3 +302,31 @@ func dedent(s string) string {
 	}
 	return strings.TrimSuffix(buf.String(), "\n")
 }
+
+func buildAliases(cmd *cobra.Command, aliasList []string) string {
+	if len(aliasList) == 0 {
+		return "No Aliases"
```

Comment:
couldn't tell from your point number 2 - `Removed "No aliases" showing up for commands without aliases` if that was just for `gh reference` or if you did not want that displayed for `gh <cmd> [<subcmd>...] --help` as well.

Reply:
Ah sorry, yes that was making `gh reference` not show `Aliases` if there were no aliases like `--help`